pg_search/src/index/search.rs

```diff
@@ -314,42 +323,46 @@ impl<'de> Deserialize<'de> for SearchIndex {
     where
         D: Deserializer<'de>,
     {
-        // A helper struct that lets us use the default serialization for most fields.
         #[derive(Deserialize)]
         struct SearchIndexHelper {
             schema: SearchIndexSchema,
             directory: WriterDirectory,
-            // An index created in an older version of pg_search may not have serialized a uuid
-            // to disk. Just use an empty string for backwards compatibility.
             #[serde(default)]
             uuid: String,
```

Comment:
Why remove these comments?

Reply:
great catch. this was removed mistakenly. restored.